grails-app/views/layouts/main.gsp

```diff
@@ -9,16 +9,32 @@
 
     <link href="${grailsApplication.config.headerAndFooter.baseURL}/css/bootstrap.min.css" rel="stylesheet" media="all" />
     <link href="${grailsApplication.config.headerAndFooter.baseURL}/css/ala-styles.css" rel="stylesheet" media="all" />
-    <asset:stylesheet src="core.css" media="all" />
-    <asset:stylesheet src="core-screen-print.css" media="all" />
+    <g:if test="${grailsApplication.config.headerAndFooter.version == '1'}">
```

Comment:
this seems to a duplication of logic in ala-bootstrap3 - or am i missing something ?

it would be nice to have this headerAndFooter.version logic in one place...

Reply:
It would be nice to update the ala-install demo playbook to to see how this `headerAndFooter.version` works. Thanks